koku/attribute/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.4 on 2019-10-22 18:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Attribute',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='The name of the attribute', max_length=255)),
+                ('value', models.TextField(null=True)),
+                ('description', models.TextField(null=True)),
+                ('updated_timestamp', models.DateTimeField(auto_now=True, null=True)),
+                ('created_timestamp', models.DateTimeField(auto_now_add=True, null=True)),
```

Comment:
What's the reason for keeping track of two dates?  I can see maybe keeping one timestamp when the field is updated.